keras_hub/src/utils/tensor_utils.py

```diff
@@ -262,6 +264,15 @@ def assert_tf_libs_installed(symbol_name):
         )
 
 
+def assert_bounding_box_support(symbol_name):
+    keras_version = re.sub(r".dev.*", "", keras.__version__)
```

Comment:
why do you need this line? `version` should handle dev release markers and other stuff here https://packaging.python.org/en/latest/discussions/versioning/